README.md

```diff
@@ -52,7 +52,18 @@ vars:
     amplitude_schema: your_schema_name 
 ```
 
-## (Optional) Step 4: Additional configurations
+## Step 4: Configure event date range
+Because of the typical volume of event data, you may want to limit this package's models to work with a more recent date range. Therefore we have added the functionality to filter records via the `date_range_start` and `date_range_end` variables within the `stg_amplitude__event` model. The default date range starts at '2020-01-01' and ends one month past the current day, but you may configure it in your root `dbt_project.yml` file:
+
+```yml
+vars:
+    amplitude:
+      date_range_start: '2022-01-01' # your start date here
+      date_range_end: '2022-12-01' # your end date here
```

Comment:
Let's make these global now that the variables are used in the source **and** the transform package.
```suggestion
    date_range_start: '2022-01-01' # your start date here
    date_range_end: '2022-12-01' # your end date here
```